### PR TITLE
Ensure edited file's attributes remain intact

### PR DESCRIPTION
**Pull request by maxchuquimia**
I have a case where I create a Swift/Beak script and run `chmod +x` on it so that I can use it as a normal command (with `#!/usr/bin/env beak run --path` too)

I found that if I `beak edit -p` the file (amazing, by the way), upon committing the changes the file is no longer executable so I need to `chmod +x` again.

I've made this change and it works, however please let me know if there's a better, more PathKit-y way that you would do this ☺️

### CODE REVIEW

**Comment by yonaskolb** [comment]
Great, thanks @maxchuquimia! Could you add a changelog entry as well?

**Comment by yonaskolb** [comment]
Resolves #11

**Comment by maxchuquimia** [comment]
@yonaskolb done :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Change Log
 
 ## Next Version
+- Fixed edit command removes file attributes #11 @maxchuquimia
```
Comment (yonaskolb) [inline CHANGELOG.md]:
```suggestion
- Fixed edit command removing file attributes #46 @maxchuquimia
```

Comment (maxchuquimia) [reply CHANGELOG.md]: Ah okay, done!

**Comment by yonaskolb** [comment]
Thanks @maxchuquimia!

**Pull request merged**